entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -1,130 +1,128 @@
 ---
-title: ASP.NET Core - New Database | Microsoft Docs
-author: rowanmiller
-ms.author: divega
-ms.date: 10/27/2016
+title: ASP.NET Core - New database with Visual Studio 2017 | Microsoft Docs
+author: rick-anderson
+ms.author: riande
+ms.author2: tdykstra
+ms.date: 04/07/2017
 ms.assetid: e153627f-f132-4c11-b13c-6c9a607addce
 ms.technology: entity-framework-core
 uid: core/get-started/aspnetcore/new-db
 ---
 
-# ASP.NET Core - New Database
+# ASP.NET Core - New database with Visual Studio 2017
 
-> [!NOTE]
-> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../../ef6/index.md).
+In this walkthrough, you will build an ASP.NET Core MVC application that performs basic data access using Entity Framework Core. You will use migrations to create the database from your model. See [Additional Resources](#additional-resources)for more Entity Framework Core tutorials.
 
-> [!IMPORTANT]
-> The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.0.0 no longer supports `project.json` or Visual Studio 2015. Everyone doing .NET Core development is encouraged to [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/) and [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+This tutorial requires [Visual Studio 2017](https://www.visualstudio.com/downloads/) and the [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1 or later.
 
-In this walkthrough, you will build an ASP.NET Core MVC application that performs basic data access using Entity Framework. You will use migrations to create the database from your model.
+Notes: 
 
-> [!TIP]
-> You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb) on GitHub.
+- You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb) on GitHub.
+- This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../../ef6/index.md).
+- The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1.x no longer supports `project.json` or Visual Studio 2015. We recommend you [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/). If you are using Visual Studio, we recommend you migrate to [Visual Studio 2017](https://www.visualstudio.com/downloads/).
```

Comment:
Great idea. I was just moving boiler-plate (BP) they have in every tutorial.  Let's flesh out new BP. We could even make the BP an include so it will be easy to upgrade next version.